docs/docs/7-migration-guide/migration-from-v7-to-v8.md

```diff
@@ -1119,6 +1119,72 @@ Remove the `allCategories` prop from `PagesPage`:
 
 :::
 
+### Add proxy for `/dam` URLs
```

Comment:
All changes can probably be done in a upgrade script. I'll implement that later when I hopefully have more time. For now I'll leave it in the guide